samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.corebot.app.insights;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.microsoft.applicationinsights.core.dependencies.apachecommons.lang3.StringUtils;
```

Comment:
Is this ok?

Reply:
isn't that the import for StringUtils?

Reply:
But you are accessing from the `applicationinsights` library, this is weird